src/scripts/build.sh

```diff
@@ -19,7 +19,11 @@ parse_tags_to_docker_arg() {
     fi
   done
 
+  # Set IFS to null to stop "," from breaking bash substitution
+  local old_ifs="$IFS"
+  local IFS=
   DOCKER_TAGS_ARG="$(eval echo $docker_arg)"
+  local IFS="$old_ifs"
```

Comment:
We don't need to save and roll back the IFS here because we are working with a `local` IFS variable in a function scope. Outside the function, the IFS will retain its original value. See [this](https://stackoverflow.com/a/50728996) for more information.

Reply:
Good point, 

Updated